RepositoryQuery.php

```diff
@@ -91,6 +91,23 @@ public static function parseSparqlResults($sparql) {
    *   The contents returned, in the $format specified.
    */
   protected function internalQuery($query, $type = 'itql', $limit = -1, $format = 'Sparql') {
+    # Patch to work around the fact that fuseki does not like hyphens
+    # in prefixes and Mulgara autodefines xsd
```

Comment:
Where did you find that Fuseki "does not like hyphens"?

Reply:
We identified that issue in testing but it has been so long that we need to recover the details.  I have other PHP queries in Drush that demonstrated the same issue.  I seem to recall it was more a call to Fuseki via that RI exhibited the issue, not Fuseki itself.

Reply:
If it's the RI code, then let's change this comment and not worry about it further. I would be frankly astonished if it were Fuseki and _no one has noticed it in a decade_!

Reply:
Agreed.  I will see if we can recover the findings, comment and document it properly.  Most of this PR is of interest only to us and not the wider Islandora community.  But the RI issue may be useful for those who are using trippi-sparql.  I don't think it was a generic issue hyphens but a very specific case for handling namespaces with hyphens via the RI. Unfortunately it was problem because some traditional Fedora namespaces contain hyphens.  Interestingly Islandora is still maintaining Tuque and there have been updates from them.